pkg/azureconstants/azure_constants.go

```diff
@@ -122,6 +122,7 @@ const (
 	VolumeAttachRequestAnnotation         = "disk.csi.azure.com/volume-attach-request"
 	VolumeDeleteRequestAnnotation         = "disk.csi.azure.com/volume-delete-request"
 	VolumeDetachRequestAnnotation         = "disk.csi.azure.com/volume-detach-request"
+	CloudDeleteVolume                     = "cloud-delete-volume"
```

Comment:
np: Let's move this up next to the `VolumeDeleteRequestAnnotation` constant since that is what it is used with.